### DISCUSSION TITLE
Integrate flet with python-vlc

### DISCUSSION THREAD

**Discussion by footballPrince** [flet-dev/flet#4116] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

Any one try to use the python-vlc library in flet app? I would like to use the python-vlc library in my flet app to display the live streaming video, anyone know how to integrate, thx!!🙏

**Comment by mfaessel-CCPS** [comment]
Hi, 

I'm also interested in examples for using python-vlc in Flet App 

Thanks !

**Comment by ndonkoHenri** [comment]
Flet has a [`Video`](https://flet.dev/docs/controls/video/) control, which you might find useful.
